ezbake/config/conf.d/web-routes.conf

```diff
@@ -2,11 +2,8 @@ web-router-service: {
     # These two should not be modified because the Puppet 4.x agent expects them to
     # be mounted at these specific paths
     "puppetlabs.services.ca.certificate-authority-service/certificate-authority-service": "/puppet-ca"
-    "puppetlabs.services.master.master-service/master-service": {master-routes:       "/puppet"
-
-                                                                 # The following controls 404 errors and should not
-                                                                 # be modified
-                                                                 invalid-in-puppet-4: "/" }
+    "puppetlabs.services.master.master-service/master-service": "/puppet"
+    "puppetlabs.services.legacy-routes.legacy-routes-service/legacy-routes-service": "/"
```

Comment:
After making this change by hand, I was able to build packages off this branch and do a successful 3x->4x agent run! :godmode: